config.analysis

```diff
@@ -42,10 +43,10 @@ climo_yr2 = 10
 yr_offset = 1849
 
 [ohc_timeseries]
-generate = 1
+generate = 0
```

Comment:
Same idea for changing logical flags-- I don't think these should be edited unless absolutely necessary.